docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
```

Comment:
What do you mean by reactivity?

Reply:
"according to a [nimble / light-weight] process that ensures quality" ?

Reply:
Reactivity = short time-to-translate, short time between new term proposed and new term translated.

I'll keep James' suggestion.